### PR TITLE
Add support Laravel 10.x & remove unused package

### PR DESCRIPTION
**Pull request by dinhquochan**
Dear,

I add support Laravel 10.x and remove `imliam/laravel-blade-helper`. Because we don't need it anymore 😅

Ref: #25

### CODE REVIEW

**Comment by stancl** [comment]
Could you expand on why that dependency is not needed anymore?

**Comment by dinhquochan** [comment]
> Could you expand on why that dependency is not needed anymore?

Because we can use directly `Blade::directive(...)`, do not need to use the above package.

File: composer.json [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
     },
     "require-dev": {
-        "orchestra/testbench": "^6.23|^7.0",
-        "nunomaduro/larastan": "^1.0",
-        "pestphp/pest": "^1.2",
-        "pestphp/pest-plugin-laravel": "^1.0",
+        "orchestra/testbench": "^6.23|^7.0|^8.0",
+        "nunomaduro/larastan": "^1.0|^2.4",
+        "pestphp/pest": "^1.2|^2.0",
+        "pestphp/pest-plugin-laravel": "^1.0|^2.0",
```
Comment (stancl) [inline composer.json:34]: I think for these dev dependencies we only need the latest version? Or do the 2.x versions of larastan & pest not support older versions of Laravel?

Comment (dinhquochan) [reply composer.json:34]: Yes. Example: Larastan drop support Laravel 8.x, Pest Laravel drop 9.x and 8.x in version 2.0 😅

Comment (stancl) [reply composer.json:34]: I see, thanks!

**Pull request merged**